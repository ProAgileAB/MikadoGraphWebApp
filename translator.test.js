/* The responsiblity of the translator is to
   translate from the parse result data structure
   to graphviz text format
 */

const { parse } = require('./parser.js')
const { translate } = require('./translator.js')

describe("translator", () => {
    it("can translate a single goal", () => {
        expect(translate(parse("Goal\n  The goal\n"))).toMatchSnapshot()
    })
})
