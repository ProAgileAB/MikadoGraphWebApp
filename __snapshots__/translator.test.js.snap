// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`translator can translate a single goal 1`] = `
"digraph G {
   end [peripheries=2, shape=oval, label=\\"The goal\\", style=filled, fillcolor=lightblue];
}"
`;
